new-pipeline/src/evo.js

```diff
@@ -324,16 +324,24 @@ $(function () {
               var channel = $("#max-channel-version").val().split("/")[0]
               var description = getDescription(metric, channel, evolutionDescription);
               var link = getDescriptionLink(metric, channel, description);
+              var useCounterLink = getUseCounterLink(metric, channel, description);
               $("#submissions-title").text(metric + " submissions");
               $("#sample-counts-title").text(metric + " sample counts");
               if (metric != description) {
                 $("#evo-caption-text").html(description);
               } else {
                 $('#evo-caption-text').text("");
+                $('#use-counter-link').text("");
```

Comment:
This should not be needed as you're already clearing or setting the link text below.